### PR TITLE
run pylint and made corrections.

### PR DESCRIPTION
**Pull request by deepgandhi**
1. removed paths import at line 57
2. removed unused variable "j" at line 88.

issue #158

### CODE REVIEW

File: minegauler/core/api.py [hunk 1]
```diff
@@ -232,8 +232,9 @@ def wrapped(*args, **kwargs):
                 try:
                     getattr(listener, func)(*args, **kwargs)
                 except Exception as e:
+                    write=f"Error occurred calling {func}() on {listener}"
                     self._logger.warning(
-                        f"Error occurred calling {func}() on {listener}"
+                        write
```
Comment (LewisGaul) [inline minegauler/core/api.py:237]: The proper way to fix this is to do
```python
self._logger.warning(f"Error occurred calling %s() on %s", func, listener)
```
as is already done in `resize_minefield()` for example (your change there should be reverted).

See https://docs.python.org/3/howto/logging.html#logging-variable-data.

**Review by LewisGaul — CHANGES_REQUESTED** [review]
Thanks, all these changes look fine except the logging changes.

In general I'd recommend using a dedicated branch for creating a PR, i.e in your fork don't raise PRs from the `dev` branch. That way it's possible to work on multiple things without having to submit it all in one PR.

File: minegauler/core/api.py (hunk 1 above)
Comment (deepgandhi) [reply minegauler/core/api.py:237]: Okay, i will look into it and resubmit it after correction.

Comment (deepgandhi) [reply minegauler/core/api.py:237]: I have made the corrections and raised new pr#181

**Pull request closed without merging**